source/_includes/common-tasks/enable_i2c.md

```diff
@@ -1,23 +1,25 @@
 ## Enable I2C
 
-Home Assistant using the Home Assistant Operating System is a managed environment, which means you can't use existing methods to enable the I2C bus on a Raspberry Pi.
+Home Assistant using the Home Assistant Operating System is a managed environment, which means you can't use existing methods to enable the I2C bus on a Raspberry Pi. In order to enable I2C devices you will have to 
+- Enable I2C for the Home Assistant Operating System 
+- Enable Raspberry Hats including the I2C integration
+- Setup I2C sensors
 
-### Step by step instructions
+### Step by step instructions to enable I2C for the Home Assistant Operating System
 
-You will need:
+#### Use SD Card/Step 1 - Access the Home Assistant Operating System boot partition
```

Comment:
This replacing of the header, removed the introduction paragraph.

Reply:
Section titles cleaned of SD card reference